libs/sdk-ui-dashboard/src/presentation/topBar/buttonBar/button/selectors.ts

```diff
@@ -20,18 +20,68 @@ import {
     selectCanEditLockedDashboardPermission,
     selectCanShareDashboardPermission,
     selectCanShareLockedDashboardPermission,
+    selectCanManageWorkspace,
+    selectCanManageAnalyticalDashboard,
+    selectSupportsHierarchicalWorkspacesCapability,
```

Comment:
The new selectors below fallback to the original edit button behavior when dashboard permissions are not enabled